tests/baselines/reference/APISample_transform.js

```diff
@@ -584,6 +584,9 @@ declare module "typescript" {
         name?: Identifier;
         body: Block | Expression;
     }
+    interface ArrowFunctionExpression extends FunctionExpression {

```

Comment:
I think this should just be extending from `Expression` and `FunctionLikeDeclaration` (since it's not a `PrimaryExpression` nor does it have a name.